src/components/Cell.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+interface CellContent {
```

Comment:
The practice is to name these interfaces or types as [ComponentName] + Props; i.e: CellProps